target_chains/ethereum/entropy_sdk/solidity/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pythnetwork/entropy-sdk-solidity",
-  "version": "2.0.0",
```

Comment:
please leave this as 2.0.0 -- we haven't released this version yet so no need to bump